.github/actions/docker-build-composite-action/action.yml

```diff
@@ -0,0 +1,65 @@
+name: 'Dynamo Container Build'
```

Comment:
Can we have the actions all just in a single folder and name them accordingly?

For example, `.github/actions/docker-build.yaml`

Reply:
I'm mistaken here, the actions are referenced by filepath and not by name inside the workflows, this is the correct way to do things.